numbagg/moving.py

```diff
@@ -38,3 +38,265 @@ def move_mean(a, window, min_count, out):
             count -= 1
 
         out[i] = asum / count if count >= min_count else np.nan
+
+
+@ndmoving(
+    [(float32[:], int64, int64, float32[:]), (float64[:], int64, int64, float64[:])]
+)
+def move_sum(a, window, min_count, out):
+    asum = 0.0
+    count = 0
+
+    for i in range(window):
+        ai = a[i]
+        if not np.isnan(ai):
+            asum += ai
+            count += 1
+        out[i] = asum if count >= min_count else np.nan
+
+    for i in range(window, len(a)):
+        ai = a[i]
+        aold = a[i - window]
+
+        ai_valid = not np.isnan(ai)
+        aold_valid = not np.isnan(aold)
+
+        if ai_valid and aold_valid:
+            asum += ai - aold
+        elif ai_valid:
+            asum += ai
+            count += 1
+        elif aold_valid:
+            asum -= aold
+            count -= 1
+
+        out[i] = asum if count >= min_count else np.nan
+
+
+# TODO: pandas doesn't use a `min_count`, which maybe makes sense, but also makes it inconsistent?
```

Comment:
could `assert min_count == 0` or perhaps `-1` to indicate disabling the `min_count` feature?